examples/process_script.py

```diff
@@ -1,142 +1,192 @@
+        variable_units = ['mGal', 'mGal', 'mGal', 'mGal', 'mGal']
+        plot_title = campaign + ' ' + flight + ': Corrections'
+        plot_name = os.path.join(outdir, campaign + '_' + flight + '_DGP_TS_corrections.png')
+        timeseries_gravity_diagnostic(results, variables, variable_units, begin_line, end_line,
+                                      plot_title, plot_name)
+
+        variables = ['abs_grav', 'corrected_grav', 'filtered_grav']
+        variable_units = ['mGal', 'mGal', 'mGal']
+        plot_title = campaign + ' ' + flight + ': Gravity'
+        plot_name = os.path.join(outdir, campaign + '_' + flight + '_DGP_TS_freeair.png')
+        timeseries_gravity_diagnostic(results, variables, variable_units, begin_line, end_line,
+                                      plot_title, plot_name)
+
+        if QC_plot:
+            # QC Segment Plot
+            variables = ['filtered_grav', 'corrected_grav', 'abs_grav']
+            variable_units = ['mGal', 'mGal', 'mGal', 'mGal']
+            plot_title = campaign + ' ' + flight + ': Gravity (segment)'
+            plot_name = os.path.join(outdir, campaign + '_' + flight + '_DGP_TS_freeair_segment.png')
+            timeseries_gravity_diagnostic(results, variables, variable_units,
+                                          QC_segment['start'], QC_segment['end'],
+                                          plot_title, plot_name)
+
+        if add_map_plots:
+            plot_title = campaign + ' ' + flight + ': Gravity'
+            plot_name = os.path.join(outdir, campaign + '_' + flight + '_DGP_mapplot_gravity.png')
+            mapplot_segment(results, 'filtered_grav',
+                         QC_segment['start'], QC_segment['end'],
+                         'mGal', plot_title, plot_name)
+    else:
+        # Temporary plots for when graph is commented out (currently OIB_P3)
+        variables = ['gravity', 'cross_accel', 'beam', 'temp']
+        variable_units = ['mGal', 'mGal', 'mGal', 'C']
+        plot_title = campaign + ' ' + flight + ': QC'
+        plot_name = os.path.join(outdir, campaign + '_' + flight + '_DGP_TS_QC.png')
+        timeseries_gravity_diagnostic(gravity, variables, variable_units, begin_line, end_line,
+                                      plot_title, plot_name)
+
+        variables = ['ell_ht', 'ortho_ht', 'lat', 'long']
+        variable_units = ['m', 'm', 'degrees', 'degrees']
+        plot_title = campaign + ' ' + flight + ': PNT'
+        plot_name = os.path.join(outdir, campaign + '_' + flight + '_DGP_TS_pnt.png')
+        timeseries_gravity_diagnostic(trajectory, variables, variable_units, begin_line, end_line,
+                                      plot_title, plot_name)
+
+        plot_title = campaign + ' ' + flight + ': Gravity'
+        plot_name = os.path.join(outdir, campaign + '_' + flight + '_DGP_mapplot_gravity.png')
+        mapplot_segment(trajectory_full, trajectory, gravity, 'gravity', 'mGal', plot_title, plot_name)
```

Comment:
Lots of string construction here.  More readable if you use `.format` or f strings. For example,
```
plot_name = os.path.join(outdir, campaign + '_' + flight + '_DGP_mapplot_gravity.png')
```
becomes
```
plot_name = os.path.join(outdir, f'{campaign}_{flight}_DGP_mapplot_gravity.png')
```